index.js

```diff
@@ -3,7 +3,6 @@ import {
   getTasksFromStorage, saveTasksToStorage, /* editTaskDescription, deleteTask, */
 } from './modules/taskModule.js';
 import renderTasks from './modules/renderModule.js';
-// import updateTaskStatus from './modules/statusModule.js'; // Import the new module
 
 // Add a new task
 const addTask = (description) => {
```

Comment:
I think it would be better to chain the addtask function, it would make it less wordy and follow the KISS and DRY rules